edit/edit.go

```diff
@@ -708,3 +744,22 @@ func parseMarkRune(cmd []rune) (rune, error) {
 	}
 	return ' ', errors.New("bad mark: " + string(cmd[i]))
 }
+
+func parseNumber(cmd []rune) (int, int, error) {
```

Comment:
Maybe a short comment to say what the two int returns are.
`// parseNumber parses and returns a positive integer. The first returned value is the parsed number, the second is the number of runes parsed.`
Or something.